### PR TITLE
Fps fix

### PR DESCRIPTION
**Pull request by marcbonnici**

### CODE REVIEW

File: wlauto/instrumentation/fps/__init__.py [hunk 1]
```diff
@@ -158,37 +158,43 @@ def validate(self):
...
-                self.device.execute('dumpsys SurfaceFlinger --latency-clear ')
-
-            self.collector = LatencyCollector(self.outfile, self.device, params or '',
-                                              self.keep_raw, self.logger, self.dumpsys_period,
-                                              runcmd, lstcmd, self.fps_method)
-        else:
+
+        use_gfxinfo = not self.force_surfaceflinger and (self.device.get_sdk_version() >= 23)
+        if use_gfxinfo and not hasattr(workload, 'package'):
+            self.logger.debug('Workload does not contain a pacakge; falling back to SurfaceFlinger...')
```
Comment (setrofim) [inline wlauto/instrumentation/fps/__init__.py]: typo: pacakge --> package

File: wlauto/workloads/manual/__init__.py [hunk 2]
```diff
@@ -58,6 +60,9 @@ class ManualWorkload(Workload):
         Parameter('view', default='SurfaceView',
                   description="""Specifies the View of the workload. This enables instruments that require a
                                  View to be specified, such as the ``fps`` instrument."""),
+        Parameter('package',
+                  description="""Specifies the package name of the workload. This enables instruments that
+                                 require a Package to be specified, such as the ``fps`` instrument."""),
```
Comment (setrofim) [inline wlauto/workloads/manual/__init__.py]: It's probably worth updating the description for this and for `'view'` above explaining that one is only required for "surfaceflinger" (pre- API level 23), and the  other only for "gfxinfo" (post- API level 23) methods of fps collection.

File: wlauto/instrumentation/fps/__init__.py (hunk 1 above)
Comment (marcbonnici) [inline wlauto/instrumentation/fps/__init__.py]: Fixed

File: wlauto/workloads/manual/__init__.py (hunk 2 above)
Comment (marcbonnici) [inline wlauto/workloads/manual/__init__.py]: Updated

**Pull request merged**